lib/fastlane/actions/unlock_keychain.rb

```diff
@@ -31,12 +39,24 @@ def self.add_keychain_to_search_list(keychain_path)
         unless keychains.include?(keychain_path)
           keychains << keychain_path
 
-          commands = []
-          commands << Fastlane::Actions.sh("security list-keychains -s #{keychains.shelljoin}", log: false)
-          commands
+          Fastlane::Actions.sh("security list-keychains -s #{keychains.shelljoin}", log: false)
+          if Helper.is_test?
+            "security list-keychains -s <add>"
```

Comment:
The problem was with unit tests. Line 42 returns:

```
security list-keychains -s security list-keychains -d user /var/folders/1q/g5575kr550j89_c74w2jwj680000gn/T/foo20160105-23189-13t5ffv
```

instead of:

```
security list-keychains -s /var/folders/1q/g5575kr550j89_c74w2jwj680000gn/T/foo20160105-23189-13t5ffv
```

This `security list-keychains -d user` was taken from line 36, where command output was split. But after thinking twice I know that I've mess it up. Instead of adjusting logic I can adjust unit tests. Please wait I'll send an update.